setup/kubernetes/azure.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Set Up an Azure Kubernetes Service Cluster
+description: Learn how to set up an AKS cluster for your Coder deployment.
+---
+This deployment guide shows you how to set up an Azure Kubernetes Service (AKS)
+cluster on which Coder can deploy.
+
+## Prerequisites
+
+Please make sure that you have the [Azure
+CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest) and
+[kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed on
+your machine.
+
+## Step 1: Create the Resource Group
+
+Create a resource group (be sure to set the **$RESOURCE_GROUP** and
+**$LOCATION** environment variables accordingly):
+
+```bash
+az group create \ 
+  --resource-group "$RESOURCE_GROUP" \ 
+  --location "$LOCATION"
+```
+
+## Step 2: Create the Azure Kubernetes Service Cluster
+
+Create the Azure Kubernetes Service Cluster (be sure to replace the placeholder
+and environment variables with the values that apply to you):
+
+```bash
+# You may have to run `az extension add --name aks-preview`
+#
+# You may also need to create a service principal manually using
+# `az ad sp create-for-rbac --skip-assignment`, then setting the
+# --service-principal and --client-secret flags
+
+CLUSTER_NAME="MY_CLUSTER_NAME" \
+LOCATION="##" SUBSCRIPTION="##" RESOURCE_GROUP="##" \
```

Comment:
I've never set Coder up on Azure before... but we should ensure this works with CVMs.

Reply:
I believe @zmcnelis-coder said there is an issue with CVMs on Azure not being supported currently? Do we want to call that out? Something to do with the kernel not being updated yet?

Reply:
Azure [requires hacky fixes](https://codercom.slack.com/archives/C014R6H5FK8/p1610561051231200) ATM